__tests__/index.test.js

```diff
@@ -264,4 +264,77 @@ describe('POST /api/articles/:article_id/comments', () => {
         expect(err.body.msg).toBe('bad request')
       })
   })
+  it('should return an error if new comment lacks params', () => {
+    const newComment = {
+      username: 'butter_bridge',
+    }
+    return request(app)
+      .post('/api/articles/1/comments')
+      .send({newComment})
+      .expect(400)
+      .then((err) => {
+        expect(err.body.msg).toBe('bad request')
+      })
+  })
+})
+
+describe('PATCH /api/articles/:article_id', () => {
```

Comment:
Would also like to see tests for the following:
Status 400, incorrect body, e.g. inc_votes property is not a number
Test for 200 if given negative number for inc_votes